### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "read ECONNRESET"

### DISCUSSION THREAD

**Discussion by zhensolid** [community/community#14376] [Copilot Conversations]
How can I solve it? thanks

**Comment by davecheney** [comment]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by VaclavK** [reply]
lots of companies are behind proxies 

I had to uninstall the preview - I may find time later to try to bypass but some form of indirect activation code (e.g. from the phone and type code back in)  or standard ports etc. would probably help lots of users - judging by quick search here

**Comment by xuhao111** [comment]
I have the same problem，how can I solve it? thanks

**Comment by eggling** [comment]
same problem here
